server/temp.txt

```diff
@@ -0,0 +1 @@
+hello2
```

Comment:
remove

Reply:
we use temp.txt in ipfs-pinner to test file submission on rudder side when the pinner is running in a docker container, otherwise the docker has to access rudder project for testing